hack/multikueue-e2e-test.sh

```diff
@@ -78,13 +84,39 @@ function startup {
     fi
 }
 
+
+#$1 - cluster name
+function istall_jobset {
+	kubectl config use-context kind-${1}
+	kubectl apply --server-side -f https://github.com/kubernetes-sigs/jobset/releases/download/$JOBSET_VERSION/manifests.yaml
+}
+
 function kind_load {
     if [ $CREATE_KIND_CLUSTER == 'true' ]
     then
         docker pull $E2E_TEST_IMAGE
         cluster_kind_load $MANAGER_KIND_CLUSTER_NAME
         cluster_kind_load $WORKER1_KIND_CLUSTER_NAME
         cluster_kind_load $WORKER2_KIND_CLUSTER_NAME 
+
+
+	# JOBSET SETUP
+	# MANAGER
+	kubectl config use-context kind-${MANAGER_KIND_CLUSTER_NAME}
```

Comment:
Any reason to do it? It might be problematic until we have the manged-by label. Until then, I don't think users should be installing it on the management cluster.

Reply:
we only add the CRD's  in the management cluster

Reply:
sgtm